quickstart_docker/quickstart.ipynb

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
Since you included a QuickStart Jupyter notebook (awesome!), perhaps add a couple command to the `readme.md` to show how to run it?

Reply:
This is a bit of a design decision and happy to do whatever works. 

- Current behavior:
    The notebook starts on its own when you run the container as it has startup.sh as it's entry point.  This means that the interactive pyspark shell will be started with jupyter lab as a driver.  The user just needs to navigate to the URL, as instructed, and their running notebook will be there, on port 8888.

- Alternative 1:
create a notebook that has a bash on dev/null entry-point and have the user start the notebook pyspark shell with jupyter lab driver, manually (launch container, cut/paste startup.sh commands, navigate to url)

- Alternative 2:
Keep the current behavior of automatic startup but add instructions per alternative 1 that has the user start the notebook manually on another port (probably 8889) since 8888 is occupie

Reply:
This is what the output would look like:

```
/mnt/c/Users/bauma/Documents/repos/delta_lake_repos/delta_fork/quickstart_docker$ docker run --rm -it -p 8888-8889:8888-8889 djup bash
[I 2022-10-20 23:04:33.540 ServerApp] jupyterlab | extension was successfully linked.
[I 2022-10-20 23:04:33.548 ServerApp] nbclassic | extension was successfully linked.
[I 2022-10-20 23:04:33.550 ServerApp] Writing Jupyter server cookie secret to /home/NBuser/.local/share/jupyter/runtime/jupyter_cookie_secret
[I 2022-10-20 23:04:33.822 ServerApp] notebook_shim | extension was successfully linked.
[I 2022-10-20 23:04:33.853 ServerApp] notebook_shim | extension was successfully loaded.
[I 2022-10-20 23:04:33.854 LabApp] JupyterLab extension loaded from /usr/local/lib/python3.8/dist-packages/jupyterlab
[I 2022-10-20 23:04:33.854 LabApp] JupyterLab application directory is /usr/local/share/jupyter/lab
[I 2022-10-20 23:04:33.858 ServerApp] jupyterlab | extension was successfully loaded.
[I 2022-10-20 23:04:33.864 ServerApp] nbclassic | extension was successfully loaded.
[I 2022-10-20 23:04:33.864 ServerApp] Serving notebooks from local directory: /opt/spark/work-dir
[I 2022-10-20 23:04:33.864 ServerApp] Jupyter Server 1.19.1 is running at:
[I 2022-10-20 23:04:33.864 ServerApp] http://07f8d8439671:8888/lab?token=f38166b1227c660ef96db8b2736fd246968a1e580653de8e
[I 2022-10-20 23:04:33.865 ServerApp]  or http://127.0.0.1:8888/lab?token=f38166b1227c660ef96db8b2736fd246968a1e580653de8e
[I 2022-10-20 23:04:33.865 ServerApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[W 2022-10-20 23:04:33.868 ServerApp] No web browser found: could not locate runnable browser.
[C 2022-10-20 23:04:33.868 ServerApp]

    To access the server, open this file in a browser:
        file:///home/NBuser/.local/share/jupyter/runtime/jpserver-7-open.html
    Or copy and paste one of these URLs:
        http://07f8d8439671:8888/lab?token=f38166b1227c660ef96db8b2736fd246968a1e580653de8e
     or http://127.0.0.1:8888/lab?token=f38166b1227c660ef96db8b2736fd246968a1e580653de8e

Reply:
Oh, we can go with the first choice per your design, please clarify this as you suggested and then I think we're good to go, eh?!

Reply:
this is done

Reply:
this is done

Reply:
👍